subsystems/motor/speedmodifiers/CapSpeedModifier.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers;
+
+
+public class CapSpeedModifier implements SpeedModifier {
+	public final double maxSpeed;
+	public final double minSpeed;
+	
+	public CapSpeedModifier(double max, double min) {
+		maxSpeed = max;
+		minSpeed = min;
+	}
+	
+	@Override
+	public double modify(double speed) {
```

Comment:
As @leijurv pointed out, although this is a perfectly fine implementation, for consistency we should make a `Util.Range` in the constructor, and then call `limitValue` on it here.